hashicorp/dev-portal
# chore: adds product meta provider to vagrant.io

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksstag-vagrant-io-opt-in-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1202702246158644/1202632199481633) 🎟️

## 🗒️ What

Adds the product meta provider to the vagrant.io layout so that the beta opt in message can render. 

## 🧪 Testing

- Select vagrant in the product switcher
- Go to a docs page and verify that the opt in message renders. 

<img width="1417" alt="Screen Shot 2022-08-08 at 4 58 52 PM" src="https://user-images.githubusercontent.com/36613477/183534387-b997c2e1-382e-417f-947b-ee29b5137cd5.png">
